app/javascript/components/api/boards-api.js

```diff
@@ -6,7 +6,23 @@ export const boardApi = {
       const {data} = await api.get(`boards/my`);
       return data;
     } catch (error) {
-      alert(error);
+      if (error.response) {
```

Comment:
We have the same `catch` block for all requests, is it possible to move it to https://github.com/cybergizer-hq/retrospective/blob/master/app/javascript/components/api/api.js#L29?
What do you think?